### ISSUE TITLE
Heltec Esp32 V3 TX non stop

### ISSUE THREAD

**Issue by nazainurin** [richonguzman/LoRa_APRS_iGate#203]
Im using latest firmware for Heltec Esp32 V3 and it TX non stop. On lcd screen appeared TX information and not back to normal. What can I do?

![IMG_20241210_225604.jpg](https://github.com/user-attachments/assets/a90fd9d9-5314-4ee7-96e8-d12d882353cd)

**Comment by richonguzman** [comment]
This is tracker firmware?

**Comment by nazainurin** [comment]
> This is tracker firmware?

yes. latest firmware for tracker. If im use for iGate, this board always restart when received the signal. But now I want to use for tracker. My board already install GPS module.

**Comment by richonguzman** [comment]
Hi, could you test it further?

**Comment by richonguzman** [comment]
this should been solved with the latest updates for Heltec boards

**Comment by nazainurin** [comment]
still same

**Comment by richonguzman** [comment]
which version of V3 is your board? V3 or V3.2 ?

you have this board flashed with Tracker or iGate firmware?

**Comment by nazainurin** [comment]
im working as tracker
*MOHD NAZAINURIN BIN MOHD NAWAWI*
*9M2OCW / 9M4BCW / 9M4GCW*
*Malaysia Amateur Radio Operator*

*No 18, Jalan Pulasan 32,*
*Taman Kota Masai,*
*81700 Pasir Gudang, *
*Johor Darul Ta'zim.*

*Email : ***@***.*** / ***@***.****
*No tel : 011-1072 2904 / 011-1214 1521*

**Comment by richonguzman** [comment]
hi, lets work it as a tracker:

which firmware version did you use?

HELTEC_V3_GPS
HELTEC_V3_TNC??

is yours new HELTEC_V3.2?
